packages/apps/plugins/plugin-deck/src/components/DeckLayout.tsx

```diff
@@ -295,69 +294,73 @@ export const DeckLayout = ({
   const { t } = useTranslation(DECK_PLUGIN);
   const { graph } = useGraph();
 
-  // TODO(wittjosiah): Finding nodes in the graph should probably not be done at the top-level of layout.
```

Comment:
doesn't seem like this has been resolved?

Reply:
👍